### PR TITLE
Code optimization

### PR DESCRIPTION
**Pull request by kirlat**
Updated to work with the latest version of components. Added an option to load components from JSDelivr CDN. Updated dependencies

### CODE REVIEW

File: src/mod.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export default function modA () {
```
Comment (balmas) [inline src/mod.js]: what's this?

File: demo/index.html [hunk 2]
```diff
@@ -20,14 +19,18 @@
 </head>
 
 <body>
-    <script src="../dist/alpheios-embedded.js"></script>
+<!--    <script src="../dist/alpheios-embedded.js"></script>-->
```
[the remark this answers is not in the document]
Comment (kirlat) [reply demo/index.html]: Will do

File: package.json [hunk 3]
```diff
@@ -35,9 +35,10 @@
     "alpheios-data-models": "github:alpheios-project/data-models",
     "alpheios-experience": "github:alpheios-project/experience",
     "alpheios-inflection-tables": "git://github.com/alpheios-project/inflection-tables.git",
-    "alpheios-node-build": "github:alpheios-project/node-build",
+    "alpheios-node-build": "github:alpheios-project/node-build#code-optimization",
```
[the remark this answers is not in the document]
Comment (kirlat) [reply package.json]: Will fix

File: src/mod.js (hunk 1 above)
Comment (kirlat) [reply src/mod.js]: Oh, that's a leftover test piece. Will remove the file

**Pull request merged**